easybuild/easyconfigs/m/matplotlib/matplotlib-2.2.5-foss-2023a-Python-2.7.18.eb

```diff
@@ -0,0 +1,70 @@
+easyblock = 'PythonBundle'
+
+name = 'matplotlib'
+version = '2.2.5'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [
+    ('pkgconf', '1.9.5'),
+]
+
+dependencies = [
+    ('Python', '2.7.18'),
+    ('numpy', '1.16.6', versionsuffix),
+    ('libpng', '1.6.39'),
+    ('freetype', '2.13.0'),
+    ('Tkinter', '%(pyver)s', versionsuffix),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('Cycler', '0.10.0', {
+        'modulename': 'cycler',
+        'source_tmpl': 'cycler-%(version)s.tar.gz',
+        'checksums': ['cd7b2d1018258d7247a71425e9f26463dfb444d411c39569972f4ce586b0c9d8'],
+    }),
+    ('kiwisolver', '1.1.0', {
+        'checksums': ['53eaed412477c836e1b9522c19858a8557d6e595077830146182225613b11a75'],
+    }),
+    ('python_dateutil', '2.9.0.post0', {
```

Comment:
We're duplicating extensions also including in `SciPy-bundle` here, so better depend on that (from #22353) rather than on `numpy`, and clean up the extensions accordingly?